sources/packages/backend/apps/api/src/route-controllers/cas-invoice-batch/cas-invoice-batch.aest.controller.ts

```diff
@@ -101,4 +119,41 @@ export class CASInvoiceBatchAESTController extends BaseController {
       throw error;
     }
   }
+
+  /**
+   * Update the approval status for a CAS invoice batch record.
+   * @param payload CAS invoice batch payload.
+   * @param casInvoiceBatchId ID of the CAS invoice batch to be updated.
+   */
+  @Roles(Role.AESTCASInvoiceBatchExpense)
+  @Patch("/:casInvoiceBatchId")
```

Comment:
Please remove the "/" from the endpoint, leaving it as `@Patch(":casInvoiceBatchId")`.